ql/src/java/org/apache/hadoop/hive/ql/optimizer/calcite/rules/views/HiveAugmentSnapshotMaterializationRule.java

```diff
@@ -141,11 +154,8 @@ public void onMatch(RelOptRuleCall call) {
     relBuilder.push(tableScan);
     List<RexNode> conds = new ArrayList<>();
     final RexNode literalHighWatermark = rexBuilder.makeLiteral(
-        mvMetaTableSnapshot.getSnapshotId(), snapshotIdType(relBuilder), false);
-    conds.add(
-        rexBuilder.makeCall(
-            SqlStdOperatorTable.LESS_THAN_OR_EQUAL,
-            ImmutableList.of(snapshotIdInputRef, literalHighWatermark)));
+        snapshotId, snapshotIdType(relBuilder.getTypeFactory()), false);
+    conds.add(rexBuilder.makeCall(SqlStdOperatorTable.LESS_THAN_OR_EQUAL, snapshotIdInputRef, literalHighWatermark));
```

Comment:
original code formatting (with new lines) was more readable

Reply:
Sorry, where were the new lines?

Reply:
````
    conds.add(
        rexBuilder.makeCall(
            SqlStdOperatorTable.LESS_THAN_OR_EQUAL,
            ImmutableList.of(snapshotIdInputRef, literalHighWatermark)));
````

Reply:
refactored